autotest/ogr/ogr_flatgeobuf.py

```diff
@@ -874,6 +874,23 @@ def test_ogr_flatgeobuf_read_invalid_geometries(filename):
         for f in lyr:
             pass
 
+###############################################################################
+# Check that we don't crash
+
+
+@pytest.mark.parametrize(
+    "filename",
+    [
+        "data/flatgeobuf/test_ogr_flatgeobuf_singlepart_mls_new.fgb",
+    ],
+)
+def test_ogr_flatgeobuf_read_singlepart_mls_new(filename):
```

Comment:
I've just tested locally that you would get an exception if you enable exceptions like this (they are disabled by default in this module). But it would be better (in the branch where it is fixed) to actually check that we get a valid geometry by inspecting a feature. I'll try to improve your other PR to do that if you can't manage to make the test suite run (normally "source ./scripts/setdevenv.sh" should do it)

```suggestion
@gdaltest.enable_exceptions()
def test_ogr_flatgeobuf_read_singlepart_mls_new(filename):
```

Reply:
I see, thanks @rouault